elastic4s-domain/src/main/scala/com/sksamuel/elastic4s/requests/searches/queries/RankFeatureQuery.scala

```diff
@@ -26,8 +26,8 @@ case class RankFeatureQuery(field: String,
 }
 
 object RankFeatureQuery {
-  case class Saturation(pivot: Option[Int])
-  case class Log(scalingFactor: Int)
-  case class Sigmoid(pivot: Int, exponent: Double)
+  case class Saturation(pivot: Option[Float])
+  case class Log(scalingFactor: Float)
+  case class Sigmoid(pivot: Float, exponent: Double)
```

Comment:
Should the exponent also be a `Float` ?

Reply:
To be completely aligned with elastic, it should be [float](https://github.com/elastic/elasticsearch-java/blob/7c46ddd904f31cab870ad5804003af29408ffe6d/java-client/src/main/java/co/elastic/clients/elasticsearch/_types/query_dsl/RankFeatureFunctionSigmoid.java#L63) instead of double. But changing this now would be a breaking change for users of this function:
![image](https://github.com/user-attachments/assets/a56a2498-d9fd-4ddf-adb7-5e61eb0b4f43)

In practice, I don't think it matters too much if it's float or a double, as I assume very little people need that kind of precision. This comment got me thinking that maybe `pivot` and `scalingFactor` should be changed to `Double` as well for consistency's sake. What do you think?

Reply:
Ok, let's leave it as is for now. I'll create a new release.